src/main/java/io/icker/factions/core/InteractionsUtil.java

```diff
@@ -22,6 +23,6 @@ public static void sync(PlayerEntity player, ItemStack itemStack, Hand hand) {
     public static void warn(PlayerEntity player, String action) {
         SoundManager.warningSound(player);
         User user = User.get(player.getUuid());
-        new Message("Cannot %s here", action).fail().send(player, !user.radar);
+        new Message(Text.translatable("factions.events.cannot_do", action)).fail().send(player, !user.radar);
```

Comment:
This does not translate the action, which is specified in english in the interaction manager